functorch/_src/config.py

```diff
@@ -23,3 +23,5 @@
 debug_joint = os.environ.get('AOT_FX_GRAPHS_JOINT', False)
 
 use_dynamic_shapes = os.getenv('AOT_DYNAMIC_SHAPES', False)
+
+static_weight_shapes = True
```

Comment:
just curious - what do you think is the end state for how we want to treat parameters <> dynamic shapes - universally on / off / depends on the model? I originally thought the idea was that parameter shapes don't really change in a model, so we always want them to be treated as static, but that sounds wrong now.

Reply:
ah, good catch, I should just remove the `nn.parameter` check.